src/components/AssignmentTexter.jsx

```diff
@@ -49,10 +50,16 @@ class AssignmentTexter extends React.Component {
 
   updateCurrentContactIndex(newIndex) {
     this.setState({
-      currentContactIndex: newIndex
+      currentContactIndex: 0
     })
   }
 
+  componentWillUpdate(nextProps, nextState) {
```

Comment:
good work-around solution for an issue we've seen as well (contant info not updating after the texter starts: https://github.com/MoveOnOrg/Spoke/issues/254).
For now, I think we want to at-most, put this behind a config option (or remove this), and then try to fix the core issue.